tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js

```diff
@@ -153,17 +160,25 @@ const DepartmentAccountsList = (properties, context) => {
             <Table.Row bold>
               <TableCell>Department Name</TableCell>
               <TableCell>Account Number</TableCell>
+              <TableCell>Account Status</TableCell>
+              <TableCell>Account Balance</TableCell>
             </Table.Row>
             {department_accounts
               .map((account) => (
-                <Table.Row key={account.account_number}
+                <Table.Row
+                  key={account.account_number}
+                  className={
+                    'AccountsUplinkTerminal__listRow--' + account.suspended
+                  }
```

Comment:
could you format the classname param like the other ones above?

Reply:
discussed on discord, keeping it consistent with security records